src/python/foglamp/configuration_manager.py

```diff
@@ -29,7 +31,8 @@
 
 _valid_type_strings = ['boolean', 'integer', 'string', 'IPv4', 'IPv6', 'X509 certificate', 'password']
```

Comment:
There is a misunderstanding: this is the list of possible configuration items, NOT the connection to the database. 
As described [here](https://docs.google.com/document/d/1QDkwb1l3XPZCTZfBVwmchgI9ay0vmfKNXUEwu1KcuOA), the database is accessible via socket with peer authentication. The connections string must be _"dbname='foglamp'"_